src/htmlize/patchtml.ml

```diff
@@ -105,6 +105,61 @@ let handle_link ~is_raw link =
                   String.concat "/" (new_link @ target);
     }
 
+open Ez_html
+open Xml
+open Xml_types
+
+(* Function to change static links found in HTML code in md files *)
+let is_edit_tag tag = tag = "a" || tag = "img"
```

Comment:
I believe there's more, try to find a list of where "hrefs" and "src" can appear. :-)